src/openrct2/interface/Theme.cpp

```diff
@@ -175,6 +175,7 @@ WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_NETWORK_STATUS),                 STR_THEMES_WINDOW_NETWORK_STATUS,                 COLOURS_1(COLOUR_LIGHT_BLUE                                                                                                    ) },
     { THEME_WC(WC_SERVER_LIST),                    STR_SERVER_LIST,                                  COLOURS_2(COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                                                   ) },
     { THEME_WC(WC_CHAT),                           STR_CHAT,                                         COLOURS_1(TRANSLUCENT(COLOUR_GREY)                                                                                             ) },
+    { THEME_WC(WC_QUICKDEMOLISH_PROMPT),           STR_THEMES_WINDOW_QUICK_DEMOLISH_PROMPT,          COLOURS_1(TRANSLUCENT(COLOUR_BORDEAUX_RED)) },
```

Comment:
In case this is kept in, please align the end `) },` with the rest of the table.